src/DeveloperGame.cs

```diff
@@ -187,33 +224,39 @@ public void CreateConnection(string connectionId, string description, Authentica
                     ((IDictionary<string, object>)arguments).Add(privilege.id + "-cansave", "on");
             }
 
-            var create_connection_response = this.Account.Client.Request($"/my/connections/create/{this.NavigationId}/{this.XSRFToken}").PostUrlEncodedAsync((object)arguments).Result;
-            var edit_connection_response = this.Account.Client.Request($"/my/connections/edit/{this.NavigationId}/{arguments.Identifier}/{this.XSRFToken}").PostUrlEncodedAsync((object)arguments).Result;
+            var create_connection_response = _client.Request($"/my/connections/create/{this.NavigationId}/{this.XSRFToken}").PostUrlEncodedAsync((object)arguments).Result;
+            var edit_connection_response = _client.Request($"/my/connections/edit/{this.NavigationId}/{arguments.Identifier}/{this.XSRFToken}").PostUrlEncodedAsync((object)arguments).Result;
         }
 
-        private BigDB GetBigDB() => new BigDB(this);
+        public Task<BigDB> LoadBigDBAsync(CancellationToken cancellationToken = default)
+            => global::PlayerIO.BigDB.LoadAsync(_client, XSRFToken, this, cancellationToken);
```

Comment:
Is there a reason for this?

Reply:
The constructor for BigDB made some asynchronous calls, and thus should've been async in the first place.

If you're referring to the lack of the async/await keywords, it's a pattern to reduce the amount of asynchronous state machines, making the binary smaller and faster (micro optimization) - see here: https://blog.stephencleary.com/2016/12/eliding-async-await.html